src/app/large-display/large-display.component.html

```diff
@@ -6,19 +6,19 @@
   </div>
 </div>
 
-<div class="bottom-bar">
-  <div class="left">
+<app-bottom-bar>
+  <div left>
     <button mat-raised-button color="accent">Quit Game</button>
   </div>
-  <div class="center">
+  <div center>
     <button mat-mini-fab color="accent"
       [disabled]="running === undefined"
       (click)="toggleTimerRunning()"
-    >
+      >
```

Comment:
```suggestion
    >
```
Just a quick indentation fix!

Reply:
Unless that was intentional on your part, in which case never mind, ignore me :slightly_smiling_face: